indico/MaKaC/webinterface/tpls/ConferenceModifFrame.tpl

```diff
@@ -1,26 +1,42 @@
 <%
 from MaKaC.webinterface.urlHandlers import UHConferenceModification
+
+event = conf.as_event
+cloned_from = event.cloned_from if event.cloned_from and not event.cloned_from.is_deleted else none
```

Comment:
are you sure lowercase `none` works in Mako? Also, I'd use `if event.cloned_from_id is not None` - that way you don't even trigger a query for the relationship in case it's None